src/styles/login.less

```diff
@@ -1,6 +1,7 @@
 // LOGIN CONTAINER
 .login-container {
   display:flex;
+  justify-content: center;
```

Comment:
Great catch on removing that gap. I noticed that the form looked off but assumed it was something I changed on my branch.